app/models/user.rb

```diff
@@ -9,13 +9,13 @@ class User < ActiveRecord::Base
   include Taxonomix
   include DirtyAssociations
   include UserTime
-  audited :except => [:last_login_on, :password, :password_hash, :password_salt, :password_confirmation]
+  audited :except => [:last_login_on, :password, :password_salt, :password_confirmation]
```

Comment:
Ok I'll try to do otherwise but I think that a part of this issue was "We need to ensure that the password change activity is logged" and that means log in Audits with changed password